drivers/led/sn32/rgb_matrix_sn32f24xb.c

```diff
@@ -235,10 +232,11 @@ static void shared_matrix_rgb_disable_output(void) {
     // Disable LED outputs on RGB channel pins
     for (uint8_t x = 0; x < LED_MATRIX_COLS; x++) {
         ATOMIC_BLOCK_FORCEON {
-#if (PWM_OUTPUT_ACTIVE_LEVEL == PWM_OUTPUT_ACTIVE_LOW)
-            writePinLow(led_col_pins[x]);
-#elif (PWM_OUTPUT_ACTIVE_LEVEL == PWM_OUTPUT_ACTIVE_HIGH)
-            writePinHigh(led_col_pins[x]);
+    // Unselect all columns before scanning the key matrix
+#if (MATRIX_OUTPUT_ACTIVE_LEVEL == PWM_OUTPUT_ACTIVE_LOW || defined(MATRIX_UNSELECT_DRIVE_HIGH))
```

Comment:
`MATRIX_UNSELECT_DRIVE_HIGH` is assumed to be defined in the `ROW2COL` scenario regardless of `RGB_OUTPUT_ACTIVE_LEVEL`. Do we need this check?

Reply:
We need `writePinHigh` at this point. 
If `MATRIX_UNSELECT_DRIVE_HIGH` is defined, then 'or' condition is true regardless of `MATRIX_OUTPUT_ACTIVE_LEVEL` and we do `writePinHigh`.
Otherwise, in my case with MATRIX_OUTPUT_ACTIVE_LEVEL = HIGH it would be `writePinLow`

So this check is needed, but I spot another error, MATRIX_OUTPUT_ACTIVE_LEVEL should be RGB_OUTPUT_ACTIVE_LEVEL, I'll fix it.